tests/vm_test.go

```diff
@@ -2062,15 +2037,7 @@ func createRunningVM(virtClient kubecli.KubevirtClient, template *v1.VirtualMach
 
 func startVM(virtClient kubecli.KubevirtClient, vm *v1.VirtualMachine) *v1.VirtualMachine {
 	By("Starting the VirtualMachine")
-	err := tests.RetryWithMetadataIfModified(vm.ObjectMeta, func(meta k8smetav1.ObjectMeta) error {
-		vm, err := virtClient.VirtualMachine(meta.Namespace).Get(context.Background(), meta.Name, k8smetav1.GetOptions{})
-		Expect(err).ToNot(HaveOccurred())
-		vm.Spec.Running = nil
-		runStrategyAlways := v1.RunStrategyAlways
-		vm.Spec.RunStrategy = &runStrategyAlways
-		_, err = virtClient.VirtualMachine(meta.Namespace).Update(context.Background(), vm, metav1.UpdateOptions{})
-		return err
-	})
+	err := patchVmRunStrategy(vm, v1.RunStrategyAlways)
```

Comment:
One liner: `Expect(patchVmRunStrategy(vm, v1.RunStrategyAlways)).To(Succeed())`

Reply:
Done here: https://github.com/kubevirt/kubevirt/pull/11948.